index.js

```diff
@@ -63,5 +67,26 @@ module.exports = opts => {
 		}
 	}
 
+	function objectEntries(object) {
+		const entries = [];
+		for (const key in object) {
```

Comment:
Use a `for-of` loop instead of a `for-in` loop.

Also `object` => `Object.keys(object)`. That way you don't have to do the `has()` check.